sdcm/tester.py

```diff
@@ -1527,6 +1527,10 @@ def run_fullscan(self, ks_cf, db_node, page_size=100000):
         cmd_select_all = 'select * from {}'
         cmd_bypass_cache = 'select * from {} bypass cache'
         cmd = random.choice([cmd_select_all, cmd_bypass_cache]).format(ks_cf)
+        if random.choice([True] * 2 + [False]):
+            cql_timeout_minutes = str(random.choice([2, 4, 8, 30, 120, 300]))
```

Comment:
Let's change it to "seconds" and not "minutes".
We saw that the timeout is per page and our paging size now is 100K rows.
So, assuming that 2 seconds sometimes may hit the timeouts and sometimes even more.

Reply:
done